homeassistant/components/alarm_control_panel/services.yaml

```diff
@@ -69,3 +69,13 @@ alarmdecoder_alarm_toggle_chime:
     code:
       description: A required code to toggle the alarm control panel chime with.
       example: 1234
+
+push_alarm_state:
```

Comment:
Should be changed to `ifttt_push_alarm_state`

Reply:
Indeed, forgot about it when renaming the service. Updating it